packages/ecc-utils-design/src/components/form/form.ts

```diff
@@ -438,7 +475,9 @@ export default class EccUtilsDesignForm extends LitElement {
     if (field.type === "array") {
       return this.renderArrayTemplate(field, newPath);
     }
-
+    if (field.type === "code") {
```

Comment:
this should be below the if statement that checks if the field is required but empty.